### PR TITLE
Feat: Gallery UI

### PR DESCRIPTION
**Pull request by tonyjojo**
## Description
Adds the basic layout and styling of the gallery and the cards.

## Task

[Notion Task Link](https://www.notion.so/d077118807034bbfb517f36c20d2a304?v=345833d9689145a2b5a197646677c93e&p=4d31a94bd08246e7a5fddd4e191aa6fc&pm=s)

## Screenshots 
dashboard

### CODE REVIEW

File: src/routes/(auth)/reset-password/+page.svelte [hunk 1]
```diff
@@ -63,8 +61,9 @@
 					<a
 						href="/login"
 						class="underline text-blue-600 hover:text-blue-800 visited:text-purple-600"
-						>Back to login page</a
 					>
```
Comment (berk245) [inline src/routes/(auth)/reset-password/+page.svelte:67]: Can we quickly talk about how to do the code formatting? I like yours way better, somehow running npm run format always returns these multi line blocks for me. like this:

<a
						href="/login"
						class="underline text-blue-600 hover:text-blue-800 visited:text-purple-600"
						>Back to login page</a
					>

**Review by berk245 — DISMISSED** [review]
Great stuff Memo! 
I am not sure how much I am into the design of cards. But that's not the main focus at the moment and it's really nice to have some layout and design. Let's discuss code formatting and db schema for tricks before we move on to next tasks

File: src/routes/(auth)/reset-password/+page.svelte (hunk 1 above)
Comment (niccoloMG) [reply src/routes/(auth)/reset-password/+page.svelte:67]: for me too! I thought it was the prettier default config for Svelte. Or at least, it is according to ChatGPT. I don't like it either way

File: src/routes/dashboard/+page.svelte [hunk 2]
```diff
@@ -1,8 +1,20 @@
 <script>
-	import { authStore } from '$lib/store/store';
+	import Card from '$components/Card.svelte';
+
+	const tricks = Array.from({ length: 20 }).map((_, i) => {
```
Comment (niccoloMG) [inline src/routes/dashboard/+page.svelte]: Nice! I didn't know about this method to easily create arrays with mock data. So convenient

**Review by niccoloMG — APPROVED** [review]

File: svelte.config.js [hunk 3]
```diff
@@ -7,7 +7,10 @@ const config = {
 		// adapter-auto only supports some environments, see https://kit.svelte.dev/docs/adapter-auto for a list.
 		// If your environment is not supported or you settled on a specific environment, switch out the adapter.
 		// See https://kit.svelte.dev/docs/adapters for more information about adapters.
-		adapter: adapter()
+		adapter: adapter(),
```
Comment (berk245) [inline svelte.config.js:10]: We have another directory called components inside lib directory, added there for easy access with $lib. This is perfectly fine as well, but let's decide on keeping one and remove the other folder. wdyt?

**Review by berk245 — COMMENTED** [review]
See the last comment about adapter and components

**Review by berk245 — COMMENTED** [review]
See the comment related to adapters and components directories

File: src/routes/(auth)/reset-password/+page.svelte (hunk 1 above)
Comment (tonyjojo) [reply src/routes/(auth)/reset-password/+page.svelte:67]: yepp, I saw that
Prettier isn't the smartest, so if you leave the text next to the ending/opening tag it won't interpret it properly and won't format it right
the only thing I did is add a line before and after the text, and then ran `npm run format`, that fixed it :pizza:

**Review by tonyjojo — COMMENTED** [review]
I combined the `components` dirs, I brought them out of `lib`, but let me know if that's not right for you
I also renamed `dashboard` for `gallery`

**Review by berk245 — APPROVED** [review]
Nice! Thanks for grouping all the components together. 🍕

**Pull request merged**